CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [0.1.4]
+* Fixes URLS for `glo-30` and `srtm_v3`
+* Includes Burst skeleton (non-functional)
+
+## [0.1.3]
+
+### Changed
+* Uses updated API dem-stitcher for square resolution cells and translation/resampling (>=2.2.0)
+* Updates dataset (patch change) from 2.0.5 to 2.0.6
+* Sort imports for updated files
+
+### Fixed
+* Uses dem-stitcher>=v2.3.0, which by default, fills in `glo-30` tiles that are missing over Armenia and Azerbaijan with the available `glo-90` tiles (upsampled).
+
```

Comment:
```suggestion
## [0.2.0]

### Changed
* Uses updated API dem-stitcher for square resolution cells and translation/resampling (>=2.2.0)
* Updates dataset (patch change) from 2.0.5 to 2.0.6
* Sort imports for updated files

### Fixed
* Uses dem-stitcher>=v2.3.0, which by default, fills in `glo-30` tiles that are missing over Armenia and Azerbaijan with the available `glo-90` tiles (upsampled).
* Uses dem-stitcher>=v2.3.1 to fix URLs for `glo-30` and `srtm_v3`
```

Since the [current release](https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/releases) version is `v0.1.2` and you've affixed the `minor` label, the next release that should be listed in the changelog is `v0.2.0`.

Reply:
And adding functionality is `minor` -- `major` is for *breaking  current functionality*:
https://semver.org/

Reply:
Thanks a million, Joe!